### ISSUE TITLE
Document and provide example of the outputs of the query tools

### ISSUE THREAD

**Issue by Remi-Gau** [neurobagel/old-query-tool#152]
Relates to the discussion from yesterday's hackathon.

Downstream tools will need to make use of the TSV outputed by the query-tool.

To make it easier it would be better if actual TSV files were stored as files in the query tool repo, so they wan be accessed by the CI of other tools.

Ideally those files should be generated by the CI of the query-tool so they are always up to date: this would allow to let downstream know of any change in the "shape" of this output because their CI would break.

Added advantage of having actual files, those examples can also be used to be shown in the [official documentation](https://www.neurobagel.org/documentation/) and not just [the wiki](https://github.com/neurobagel/documentation/wiki/Query-Tool#example-data).

## Steps for this:

- [x] https://github.com/neurobagel/neurobagel_examples/issues/2
- [x] https://github.com/neurobagel/bagel-cli/issues/187
-----
- [x] https://github.com/neurobagel/neurobagel_examples/issues/3
- [x] https://github.com/neurobagel/documentation/issues/27

**Comment by Remi-Gau** [comment]
@surchs

**Comment by surchs** [comment]
That sounds good @Remi-Gau, just to be clear here you want there to be an up to date example .tsv file inside the query tool repo and then have a link from the docs to these files, correct?

**Comment by Remi-Gau** [comment]
yes pretty much

you could even render the actual example files in the docs AND have a link

but in general, have in the repo up to date TSVs that match the actual output of the query tool

**Comment by surchs** [comment]
@neurobagel/dev. I think we need to split this into at least two parts:

1. Have queryable test endpoint / on-demand API+graph inside the CI
2. write workflow to query the endpoint, save the results and store them publicly. 

Any further splits?

**Comment by alyssadai** [comment]
@surchs since we discussed creating these examples manually for now but moving them to a central location, the issue description has been updated with relevant sub-issues. Feel free to add anything you think is missing :)

**Comment by alyssadai** [comment]
I think this issue can be closed now as example query tool outputs have been added to https://github.com/neurobagel/neurobagel_examples and referenced in the Neurobagel documentation [here](https://neurobagel.org/query_tool/#downloading-query-results).

@rmanaem do you think it'd be worth updating the query tool README to link to docs section with the example outputs?

If not, I think we can go ahead and close this.

**Comment by rmanaem** [comment]
Yea I think it'd be a good idea. I'll take care of it.
Close this once #234 is merged.

**Comment by alyssadai** [comment]
Closing as all subtasks are complete
